studies/templatetags/describe_documents.py

```diff
@@ -0,0 +1,83 @@
+from django import template
+from studies.models import Documents, Study
```

Comment:
It's customary to order imports thusly:
standard library
<whitespace>
libraries
<whitespace>
in-project code

Not really a big issue, but I noticed I was a tad confused when trying to find the source of give_name

Reply:
Good point, I usually appreciate the ordering as well! This was just laziness / forgetfulness on my part